bones/bone.py

```diff
@@ -715,32 +715,39 @@ def mergeFrom(self, valuesCache, boneName, otherSkel):
 			return
 		valuesCache[boneName] = copy.deepcopy(otherSkel.valuesCache.get(boneName, None))
 
-	def setBoneValue(self, skel, boneName, value, append, *args, **kwargs):
+	def setBoneValue(self, skel: 'SkeltonInstance', boneName: str, value: Any, append: bool,
```

Comment:
```suggestion
	def setBoneValue(self, skel: 'SkeletonInstance', boneName: str, value: Any, append: bool,
```

As already noted in the [last review](https://github.com/viur-framework/viur-core/pull/123#discussion_r479362994), there is a typo.